probables/blooms/expandingbloom.py

```diff
@@ -11,7 +11,7 @@
 from ..exceptions import RotatingBloomFilterError
 from ..hashes import HashFuncT, HashResultsT, KeyT
 from ..utilities import is_valid_file
-from .bloom import BloomFilter
+from .bloom import BloomFilter, HashFuncT
```

Comment:
Not sure why `HashFuncT` is imported here as it is pulled in from the `hashes` utility file.

Reply:
Fixed.